pyathena/load_sim.py

```diff
@@ -236,13 +363,13 @@ def func(num):
         if self.fname.endswith('vtk'):
             if self.load_method == 'pyathena':
                 self.ds = AthenaDataSet(self.fname, units=self.u, dfi=self.dfi)
-                self.domain = self.ds.domain
+                self._domain = self.ds.domain
```

Comment:
Looks like LoadSim._domain is already set from the parameter file by calling self._get_domain_from_par. Is it intended to be overwritten here?

Reply:
Ah, yes. `domain` returned from `self._get_domain_from_par` does not contain time information (`time` is set to `None`), while `domain` updated from reading vtk should have `time` in code unit.

Reply:
Although it may be valid to set time = None for `s` (LoadSim instance, which does not have specific `num`), given that `s.ds` would have time in it.